zeitgeschichte-wien.csl

```diff
@@ -0,0 +1,393 @@
+            <date-part name="year"/>
+          </date>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pages">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper chapter" match="any">
+        <text variable="page"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="URL">
+        <group delimiter=" " prefix=" (" suffix=")">
+          <text term="accessed" text-case="lowercase"/>
+          <date variable="accessed">
+            <date-part name="day" form="numeric" suffix="."/>
+            <date-part name="month" form="numeric" prefix=" " suffix="."/>
+            <date-part name="year" prefix=" "/>
+          </date>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="citation-locator">
+    <text variable="locator"/>
+  </macro>
+  <citation et-al-subsequent-min="6" et-al-subsequent-use-first="5" disambiguate-add-names="true">
+    <layout suffix="." delimiter="; ">
+      <choose>
+        <if position="ibid-with-locator">
+          <text term="ibid" text-case="capitalize-first" suffix="."/>
+          <text macro="citation-locator" prefix=", "/>
+        </if>
+        <else-if position="ibid">
+          <text term="ibid" text-case="capitalize-first" suffix="."/>
+        </else-if>
+        <else-if position="subsequent">
+          <choose>
+            <if type="motion_picture">
+              <text variable="title" form="long" font-style="italic"/>
+              <text prefix=", Regie: " macro="contributors-long"/>
+              <group prefix=", " delimiter=" ">
+                <text variable="publisher-place"/>
+                <date variable="issued">
+                  <date-part name="year"/>
+                </date>
+              </group>
+            </if>
+            <else>
+              <text macro="contributors-short" suffix=", "/>
+              <text macro="title-short"/>
+              <text prefix=", " macro="citation-locator"/>
+            </else>
+          </choose>
+        </else-if>
+        <else>
+          <choose>
+            <if type="motion_picture">
+              <text variable="title" form="long" font-style="italic"/>
+              <text prefix=", Regie: " macro="contributors-long"/>
+              <group prefix=", " delimiter=" ">
+                <text variable="publisher-place"/>
+                <date variable="issued">
+                  <date-part name="year"/>
+                </date>
+              </group>
+            </if>
+            <else>
+              <text macro="contributors-long" suffix=", "/>
+              <text macro="title-long"/>
+              <text prefix=", " macro="container"/>
+              <text macro="issued"/>
+              <text prefix=", " macro="pages"/>
+              <choose>
+                <if variable="page" type="chapter" match="all">
+                  <text prefix=", hier " macro="citation-locator"/>
```

Comment:
I'm not seeing the ", hier" for indidividual pages in the journal. Are you sure that's correct?